src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -557,7 +557,7 @@ protected override void Dispose(bool disposing)
             Task.Factory.StartNew(
                 s => {
                     var whrs = (WinHttpRequestState)s;
-                    whrs.Handler.StartRequest(whrs);
+                    _ = whrs.Handler.StartRequest(whrs);
```

Comment:
This might be a naive question, but I've never seen this syntax:

```c#
_ = whrs.Handler.StartRequest(whrs);
```

What is it? What does the "_" (underscore) mean?

Reply:
It's a "discard".
https://docs.microsoft.com/en-us/dotnet/csharp/discards

Reply:
Thanks!